src/Pages/Channels.razor

```diff
@@ -93,7 +92,27 @@
                         @context.Status.ToString("G")
                     </DisplayTemplate>
                 </DataGridColumn>
-
+                <DataGridColumn TItem="Channel" Field="@nameof(Channel.OpenedWithId)" Caption="Opened with" CustomFilter="OnWalletFilter" Filterable="true" Displayable="@IsColumnVisible(ChannelsColumnName.OpenedWith)">
+                    <FilterTemplate>
+                        <Select TValue="int" SelectedValue="@(_filterWalletId)" SelectedValueChanged="@(value => { _filterWalletId = value; context.TriggerFilterChange(_filterWalletId); })">
+                            <SelectItem Value="@(0)">All</SelectItem>
+                            @foreach (var wallet in _wallets)
+                            {
+                                <SelectItem Value="@(wallet.Id)">@wallet.Name</SelectItem>
+                            }
+                        </Select>
+                    </FilterTemplate>
+                    <DisplayTemplate>
+                        @{
+                            Wallet wallet = null;
+                            if (context.OpenedWithId != null)
+                            {
+                                wallet = Task.Run(() => WalletRepository.GetById((int)context.OpenedWithId)).Result;
+                            }
+                            @(wallet == null ? "Unknown" : wallet.Name)
```

Comment:
I wonder how much will this make the channels view slower, its async but maybe slows down, we already have the problem with the balance, worth researching